Dockerfile

```diff
@@ -53,8 +61,7 @@ RUN echo "deb http://ftp.debian.org/debian $(sed -n 's/^VERSION=.*(\(.*\)).*/\1/
         -o /usr/local/bin/wp https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar \
         -o /etc/bash_completion.d/wp-cli https://raw.githubusercontent.com/wp-cli/wp-cli/master/utils/wp-completion.bash \
     && a2enconf fqdn \
-    && a2enmod rewrite expires \
-    && service apache2 restart
```

Comment:
Isn't it needed anymore?

Reply:
Wasn't ever needed in the first place. We start apache at the end of `run.sh`.

Reply:
Makes total sense. LGTM!